diff --git a/src/po/gvim.desktop.in b/src/po/gvim.desktop.in
index 7d5234851..b547598e8 100644
--- a/src/po/gvim.desktop.in
+++ b/src/po/gvim.desktop.in
@@ -10,7 +10,6 @@ Comment=Edit text files
 # The translations should come from the po file. Leave them here for now, they will
 # be overwritten by the po file when generating the desktop.file!
 GenericName[da]=Teksteditor
-GenericName[eo]=Tekstoredaktilo
 GenericName[fr]=Éditeur de texte
 GenericName[pl]=Edytor tekstu
 GenericName[is]=Ritvinnsluforrit
@@ -29,7 +28,6 @@ Comment[da]=Rediger tekstfiler
 Comment[el]=Επεξεργασία αρχείων κειμένου
 Comment[en_CA]=Edit text files
 Comment[en_GB]=Edit text files
-Comment[eo]=Redakti tekstajn dosierojn
 Comment[es]=Edita archivos de texto
 Comment[et]=Redigeeri tekstifaile
 Comment[eu]=Editatu testu-fitxategiak
