diff -up vim74/runtime/ftplugin/spec.vim.rfc822 vim74/runtime/ftplugin/spec.vim
--- vim74/runtime/ftplugin/spec.vim.rfc822	2016-04-13 12:31:09.856418963 +0200
+++ vim74/runtime/ftplugin/spec.vim	2016-04-13 12:32:22.091802791 +0200
@@ -113,7 +113,10 @@ if !exists("*s:SpecChangelog")
 			endif
 		endif
 		if (chgline != -1)
+			let tmptime = v:lc_time
+			language time C
 			let parsed_format = "* ".strftime(format)." - ".ver."-".rel
+			execute "language time" tmptime
 			let release_info = "+ ".name."-".ver."-".rel
 			let wrong_format = 0
 			let wrong_release = 0
