diff -up vim80/src/gui.c.blinkoff-loop vim80/src/gui.c
--- vim80/src/gui.c.blinkoff-loop	2018-01-19 12:08:46.503427723 +0100
+++ vim80/src/gui.c	2018-01-19 12:09:24.542072239 +0100
@@ -1121,9 +1121,6 @@ gui_update_cursor(
 	gui_mch_set_blinking(shape->blinkwait,
 			     shape->blinkon,
 			     shape->blinkoff);
-	if (shape->blinkwait == 0 || shape->blinkon == 0
-						       || shape->blinkoff == 0)
-	    gui_mch_stop_blink();
 #ifdef FEAT_TERMINAL
 	if (shape_bg != INVALCOLOR)
 	{
