--- vim74/src/os_unix.h	2016-04-12 13:30:07.360022178 +0200
+++ vim74_patched/src/os_unix.h	2016-04-13 10:41:08.780150951 +0200
@@ -222,7 +222,7 @@
  * Unix system-dependent file names
  */
 #ifndef SYS_VIMRC_FILE
-# define SYS_VIMRC_FILE "/etc/vimrc"
+# define SYS_VIMRC_FILE "/etc/virc"
 #endif
 #ifndef SYS_GVIMRC_FILE
 # define SYS_GVIMRC_FILE "/etc/gvimrc"
@@ -266,23 +266,23 @@
 
 #ifndef USR_VIMRC_FILE
 # ifdef VMS
-# define USR_VIMRC_FILE  "sys$login:.vimrc"
+# define USR_VIMRC_FILE  "sys$login:.virc"
 # else
-#  define USR_VIMRC_FILE "$HOME/.vimrc"
+#  define USR_VIMRC_FILE "$HOME/.virc"
 # endif
 #endif
 
 
 #if !defined(USR_VIMRC_FILE2)
 # ifdef VMS
-#  define USR_VIMRC_FILE2	"sys$login:vimfiles/vimrc"
+#  define USR_VIMRC_FILE2	"sys$login:vimfiles/virc"
 # else
-#   define USR_VIMRC_FILE2	"~/.vim/vimrc"
+#   define USR_VIMRC_FILE2	"~/.vim/virc"
 # endif
 #endif
 
 #if !defined(USR_VIMRC_FILE3) && defined(VMS)
-# define USR_VIMRC_FILE3 "sys$login:_vimrc"
+# define USR_VIMRC_FILE3 "sys$login:_virc"
 #endif
 
 #ifndef USR_GVIMRC_FILE
@@ -329,7 +329,7 @@
 #endif
 
 #ifndef VIMRC_FILE
-# define VIMRC_FILE	".vimrc"
+# define VIMRC_FILE	".virc"
 #endif
 
 #ifdef FEAT_GUI
--- vim74/src/ex_cmds2.c	2016-04-08 11:00:52.000000000 +0200
+++ vim74_patched/src/ex_cmds2.c	2016-04-13 10:52:24.713437801 +0200
@@ -3696,7 +3696,7 @@
 	 */
 	p = gettail(fname_exp);
 	if ((*p == '.' || *p == '_')
-		&& (STRICMP(p + 1, "vimrc") == 0
+		&& (STRICMP(p + 1, "virc") == 0
 		    || STRICMP(p + 1, "gvimrc") == 0
 		    || STRICMP(p + 1, "exrc") == 0))
 	{
