diff -up vim73/src/testdir/Makefile.test83 vim73/src/testdir/Makefile
--- vim73/src/testdir/Makefile.test83	2012-11-13 17:51:38.493046031 +0100
+++ vim73/src/testdir/Makefile	2012-11-13 17:51:52.216052356 +0100
@@ -26,7 +26,7 @@ SCRIPTS = test1.out test2.out test3.out
 		test64.out test65.out test66.out test67.out test68.out \
 		test69.out test70.out test71.out test72.out test73.out \
 		test74.out test75.out test76.out test77.out test78.out \
-		test79.out test80.out test81.out test82.out test83.out \
+		test79.out test80.out test81.out test82.out \
 		test84.out test85.out test86.out test87.out
 
 SCRIPTS_GUI = test16.out
