diff -up vim72/runtime/autoload/netrw.vim.elinksparm vim72/runtime/autoload/netrw.vim
--- vim72/runtime/autoload/netrw.vim.elinksparm	2009-12-03 13:08:10.000000000 +0100
+++ vim72/runtime/autoload/netrw.vim	2009-12-03 13:08:27.000000000 +0100
@@ -88,7 +88,7 @@ if !exists("g:netrw_http_cmd")
  if executable("elinks")
   let g:netrw_http_cmd = "elinks"
   if !exists("g:netrw_http_xcmd")
-   let g:netrw_http_xcmd= "-dump >"
+   let g:netrw_http_xcmd= "-source >"
   endif
  elseif executable("links")
   let g:netrw_http_cmd = "links"
