diff -up vim71/runtime/filetype.vim.erlang vim71/runtime/filetype.vim
--- vim71/runtime/filetype.vim.erlang	2008-02-25 17:50:50.000000000 +0100
+++ vim71/runtime/filetype.vim	2008-02-25 17:51:17.000000000 +0100
@@ -595,7 +595,7 @@ func! s:FTe()
 endfunc
 
 " ERicsson LANGuage
-au BufNewFile,BufRead *.erl			setf erlang
+au BufNewFile,BufRead *.erl,*hrl			setf erlang
 
 " Elm Filter Rules file
 au BufNewFile,BufRead filter-rules		setf elmfilt
