diff -up vim-upstream/src/ex_getln.c.ftbfs-ccline vim-upstream/src/ex_getln.c
--- vim-upstream/src/ex_getln.c.ftbfs-ccline	2017-01-09 10:05:09.306922726 +0100
+++ vim-upstream/src/ex_getln.c	2017-01-09 10:06:31.846267643 +0100
@@ -212,7 +212,7 @@ getcmdline(
 #endif
     expand_T	xpc;
     long	*b_im_ptr = NULL;
-#if defined(FEAT_WILDMENU) || defined(FEAT_EVAL) || defined(FEAT_SEARCH_EXTRA)
+#if defined(FEAT_WILDMENU) || defined(FEAT_EVAL) || defined(FEAT_SEARCH_EXTRA) || defined(FEAT_CMDWIN)
     /* Everything that may work recursively should save and restore the
      * current command line in save_ccline.  That includes update_screen(), a
      * custom status line may invoke ":normal". */
