diff -up vim80/src/proto/beval.pro.beval-pro vim80/src/proto/beval.pro
--- vim80/src/proto/beval.pro.beval-pro	2017-11-20 18:59:12.364156610 +0100
+++ vim80/src/proto/beval.pro	2017-11-20 18:59:50.926848275 +0100
@@ -0,0 +1,6 @@
+/* beval.c */
+int get_beval_info(BalloonEval *beval, int getword, win_T **winp, linenr_T *lnump, char_u **textp, int *colp);
+void post_balloon(BalloonEval *beval, char_u *mesg, list_T *list);
+int can_use_beval(void);
+void general_beval_cb(BalloonEval *beval, int state);
+/* vim: set ft=c : */
