diff -up vim74/runtime/syntax/sshconfig.vim.kh vim74/runtime/syntax/sshconfig.vim
--- vim74/runtime/syntax/sshconfig.vim.kh	2016-01-12 14:13:15.532558597 +0100
+++ vim74/runtime/syntax/sshconfig.vim	2016-01-12 14:16:51.039800172 +0100
@@ -69,8 +69,8 @@ syn keyword sshconfigSysLogFacility DAEM
 syn keyword sshconfigSysLogFacility LOCAL2 LOCAL3 LOCAL4 LOCAL5 LOCAL6 LOCAL7
 syn keyword sshconfigAddressFamily  inet inet6
 
-syn match   sshconfigIPQoS	"af1[1234]"
-syn match   sshconfigIPQoS	"af2[23]"
+syn match   sshconfigIPQoS	"af1[123]"
+syn match   sshconfigIPQoS	"af2[123]"
 syn match   sshconfigIPQoS	"af3[123]"
 syn match   sshconfigIPQoS	"af4[123]"
 syn match   sshconfigIPQoS	"cs[0-7]"
@@ -106,6 +106,10 @@ syn keyword sshconfigMatch canonical exe
 syn keyword sshconfigKeyword AddressFamily
 syn keyword sshconfigKeyword BatchMode
 syn keyword sshconfigKeyword BindAddress
+syn keyword sshconfigKeyword CanonicalDomains
+syn keyword sshconfigKeyword CanonicalizeFallbackLocal
+syn keyword sshconfigKeyword CanonicalizeHostname
+syn keyword sshconfigKeyword CanonicalizeMaxDots
 syn keyword sshconfigKeyword ChallengeResponseAuthentication
 syn keyword sshconfigKeyword CheckHostIP
 syn keyword sshconfigKeyword Cipher
@@ -145,6 +149,8 @@ syn keyword sshconfigKeyword HostbasedKe
 syn keyword sshconfigKeyword IPQoS
 syn keyword sshconfigKeyword IdentitiesOnly
 syn keyword sshconfigKeyword IdentityFile
+syn keyword sshconfigKeyword IgnoreUnknown
+syn keyword sshconfigKeyword IPQoS
 syn keyword sshconfigKeyword KbdInteractiveAuthentication
 syn keyword sshconfigKeyword KbdInteractiveDevices
 syn keyword sshconfigKeyword KexAlgorithms
diff -up vim74/runtime/syntax/sshdconfig.vim.kh vim74/runtime/syntax/sshdconfig.vim
--- vim74/runtime/syntax/sshdconfig.vim.kh	2016-01-12 14:17:06.278889344 +0100
+++ vim74/runtime/syntax/sshdconfig.vim	2016-01-12 14:20:12.737980416 +0100
@@ -65,8 +65,8 @@ syn keyword sshdconfigSysLogFacility LOC
 
 syn keyword sshdconfigCompression    delayed
 
-syn match   sshdconfigIPQoS	"af1[1234]"
-syn match   sshdconfigIPQoS	"af2[23]"
+syn match   sshdconfigIPQoS	"af1[123]"
+syn match   sshdconfigIPQoS	"af2[123]"
 syn match   sshdconfigIPQoS	"af3[123]"
 syn match   sshdconfigIPQoS	"af4[123]"
 syn match   sshdconfigIPQoS	"cs[0-7]"
@@ -109,6 +109,7 @@ syn keyword sshdconfigKeyword AllowGroup
 syn keyword sshdconfigKeyword AllowStreamLocalForwarding
 syn keyword sshdconfigKeyword AllowTcpForwarding
 syn keyword sshdconfigKeyword AllowUsers
+syn keyword sshdconfigKeyword AuthenticationMethods
 syn keyword sshdconfigKeyword AuthorizedKeysFile
 syn keyword sshdconfigKeyword AuthorizedKeysCommand
 syn keyword sshdconfigKeyword AuthorizedKeysCommandUser
@@ -126,12 +127,14 @@ syn keyword sshdconfigKeyword DenyUsers
 syn keyword sshdconfigKeyword ForceCommand
 syn keyword sshdconfigKeyword GSSAPIAuthentication
 syn keyword sshdconfigKeyword GSSAPICleanupCredentials
+syn keyword sshdconfigKeyword GSSAPIEnablek5users
 syn keyword sshdconfigKeyword GSSAPIKeyExchange
 syn keyword sshdconfigKeyword GSSAPIStoreCredentialsOnRekey
 syn keyword sshdconfigKeyword GSSAPIStrictAcceptorCheck
 syn keyword sshdconfigKeyword GatewayPorts
 syn keyword sshdconfigKeyword HostCertificate
 syn keyword sshdconfigKeyword HostKey
+syn keyword sshdconfigKeyword HostKeyAgent
 syn keyword sshdconfigKeyword HostKeyAlgorithms
 syn keyword sshdconfigKeyword HostbasedAcceptedKeyTypes
 syn keyword sshdconfigKeyword HostbasedAuthentication
@@ -144,6 +147,7 @@ syn keyword sshdconfigKeyword KerberosAu
 syn keyword sshdconfigKeyword KerberosGetAFSToken
 syn keyword sshdconfigKeyword KerberosOrLocalPasswd
 syn keyword sshdconfigKeyword KerberosTicketCleanup
+syn keyword sshdconfigKeyword KerberosUseKuserok
 syn keyword sshdconfigKeyword KexAlgorithms
 syn keyword sshdconfigKeyword KeyRegenerationInterval
 syn keyword sshdconfigKeyword ListenAddress
