--- vim64/runtime/doc/tags.rhcvim	2005-10-15 16:00:26.000000000 +0200
+++ vim64/runtime/doc/tags	2005-12-20 12:08:06.000000000 +0100
@@ -4033,6 +4033,7 @@
 byte2line()	eval.txt	/*byte2line()*
 bzip2	pi_gzip.txt	/*bzip2*
 c	change.txt	/*c*
+c-support	csupport.txt	/*c-support*
 c-syntax	syntax.txt	/*c-syntax*
 c.vim	syntax.txt	/*c.vim*
 cW	change.txt	/*cW*
@@ -4278,6 +4279,41 @@
 csqf	if_cscop.txt	/*csqf*
 cst	if_cscop.txt	/*cst*
 csto	if_cscop.txt	/*csto*
+csupport	csupport.txt	/*csupport*
+csupport-comm	csupport.txt	/*csupport-comm*
+csupport-comm-1	csupport.txt	/*csupport-comm-1*
+csupport-comm-2	csupport.txt	/*csupport-comm-2*
+csupport-comm-3	csupport.txt	/*csupport-comm-3*
+csupport-comm-4	csupport.txt	/*csupport-comm-4*
+csupport-comm-5	csupport.txt	/*csupport-comm-5*
+csupport-comm-6	csupport.txt	/*csupport-comm-6*
+csupport-comm-7	csupport.txt	/*csupport-comm-7*
+csupport-custom	csupport.txt	/*csupport-custom*
+csupport-custom-1	csupport.txt	/*csupport-custom-1*
+csupport-custom-2	csupport.txt	/*csupport-custom-2*
+csupport-dictionary	csupport.txt	/*csupport-dictionary*
+csupport-hotkeys	csupport.txt	/*csupport-hotkeys*
+csupport-idioms	csupport.txt	/*csupport-idioms*
+csupport-proto	csupport.txt	/*csupport-proto*
+csupport-release-notes	csupport.txt	/*csupport-release-notes*
+csupport-run	csupport.txt	/*csupport-run*
+csupport-run-1	csupport.txt	/*csupport-run-1*
+csupport-run-2	csupport.txt	/*csupport-run-2*
+csupport-run-3	csupport.txt	/*csupport-run-3*
+csupport-run-4	csupport.txt	/*csupport-run-4*
+csupport-run-5	csupport.txt	/*csupport-run-5*
+csupport-run-6	csupport.txt	/*csupport-run-6*
+csupport-run-7	csupport.txt	/*csupport-run-7*
+csupport-run-8	csupport.txt	/*csupport-run-8*
+csupport-snippets	csupport.txt	/*csupport-snippets*
+csupport-stat	csupport.txt	/*csupport-stat*
+csupport-taglist	csupport.txt	/*csupport-taglist*
+csupport-tempfiles	csupport.txt	/*csupport-tempfiles*
+csupport-tempfiles-1	csupport.txt	/*csupport-tempfiles-1*
+csupport-tempfiles-2	csupport.txt	/*csupport-tempfiles-2*
+csupport-usage-gvim	csupport.txt	/*csupport-usage-gvim*
+csupport-windows	csupport.txt	/*csupport-windows*
+csupport.txt	csupport.txt	/*csupport.txt*
 csverb	if_cscop.txt	/*csverb*
 ctags	tagsrch.txt	/*ctags*
 ctags-gone	version6.txt	/*ctags-gone*
@@ -4857,7 +4893,6 @@
 hebrew.txt	hebrew.txt	/*hebrew.txt*
 help	various.txt	/*help*
 help-context	help.txt	/*help-context*
-help-tags	tags	1
 help-translated	various.txt	/*help-translated*
 help-xterm-window	various.txt	/*help-xterm-window*
 help.txt	help.txt	/*help.txt*
--- vim64/cvim/plugin/c.vim.rhcvim	2005-09-03 12:00:40.000000000 +0200
+++ vim64/cvim/plugin/c.vim	2005-12-20 12:09:37.000000000 +0100
@@ -48,7 +48,8 @@
 	"
 else
 	"
-	let s:root_dir	  = $HOME.'/.vim/'
+"	let s:root_dir	  = $HOME.'/.vim/'
+	let s:root_dir	  = $VIMRUNTIME.'/'
   let s:escfilename = ' \%#[]'
 	"
 endif
@@ -64,7 +65,7 @@
 "
 "  Modul global variables (with default values) which can be overridden.
 "
-let s:C_AuthorName     = ""
+let s:C_AuthorName     = "read README.csupport to learn how to set defaults"
 let s:C_AuthorRef      = ""
 let s:C_Email          = ""
 let s:C_Company        = ""
@@ -2465,12 +2466,12 @@
 "  Automated header insertion
 "------------------------------------------------------------------------------
 "
-if has("autocmd")
-	autocmd BufNewFile  *.c,*.cc,*.cxx,*.c++,*.C,*.H,*.hh,*.hxx,*.hpp,*.moc,*.tcc,*.inl
-				\		call C_CommentTemplates('cheader')
-	autocmd BufNewFile  *.hh,*.hxx,*.hpp
-				\		call C_CommentTemplates('hheader')
-endif " has("autocmd")
+" if has("autocmd")
+" 	autocmd BufNewFile  *.c,*.cc,*.cxx,*.c++,*.C,*.H,*.hh,*.hxx,*.hpp,*.moc,*.tcc,*.inl
+" 				\		call C_CommentTemplates('cheader')
+" 	autocmd BufNewFile  *.hh,*.hxx,*.hpp
+" 				\		call C_CommentTemplates('hheader')
+" endif " has("autocmd")
 "
 "=====================================================================================
 " vim: set tabstop=2 shiftwidth=2: 
